### ISSUE TITLE
Player and Season Change

### ISSUE THREAD

**Issue by ItzNuggetz** [rhenwinch/Flixclusive#86]
### Steps to reproduce

1. The player response time is high like it keeps buffering after every second a content plays and this is for all even though I have a good Internet connection . 
2. The player shows up with other website url  and keeps displaying over the content for example : slots lights.net. This occurred while watching inside out 2.
3. The quality preference selection doesn't help with inside out 2 as the quality is pretty low could be the reason as it's a recent theatre release. 
4. The Seasons doesn't change even after selecting in the drop down. 

I'm on the latest release. Would be great if the issues gets fixed, thanks!

![Screenshot_20240624_021419](https://github.com/rhenwinch/Flixclusive/assets/166253246/bfe22b4d-ecfb-4553-b1a7-7cb5445de817)
![Screenshot_20240624_021405](https://github.com/rhenwinch/Flixclusive/assets/166253246/5d9d6a16-460e-4630-ab9a-234b6a15a485)
![Screenshot_20240624_020418](https://github.com/rhenwinch/Flixclusive/assets/166253246/037e8c21-0ef1-4023-8b83-1ea866e821f7)

### Expected behavior

The player should be able to play the videos in less time with good quality overall. The seasons and episodes should change based on the drop down selection made. 

### Actual behavior

The player response time is high and seasons don't change even after a drop down selection. 

### Flixclusive version

1.4.0

### Android version

Android 13

### Device

Honor 90

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the **[latest version](https://github.com/rhenwinch/Flixclusive/releases/latest)**.

**Comment by rhenwinch** [comment]
To sum up what I'm gonna say: this is basically a provider-related issue and this is out of the scope for the app.

> 1. The player response time is high like it keeps buffering after every second a content plays and this is for all even though I have a good Internet connection .

 Good internet connection is _not the only thing_ you need for seamless playback. You must also _think about_ how fast the servers of your provider are. So in layman's terms, even if your internet is _fast_, you can still have problems if the streaming service's servers are _slow_.

> 2. The player shows up with other website url and keeps displaying over the content for example : slots lights.net. This occurred while watching inside out 2.
> 3. The quality preference selection doesn't help with inside out 2 as the quality is pretty low could be the reason as it's a recent theatre release.

You already mentioned that the movie you're watching is a _very recent_ one, so most providers will probably give you a _CAMRip_ version (just like the image you pasted). That could also be the reason why a lot of shenanigans (_ads_) could be included on the media playback. 

In general, we cannot rely on providers to get _HQ/HD_ streams immediately for newly premiered movies like this one. Give it some time.

---
> 4. The Seasons doesn't change even after selecting in the drop down.

May I ask, what tv show are you trying to see?

**Comment by xeamaic** [comment]
4.  The Seasons doesn't change even after selecting in the drop down.

Hi @rhenwinch  I've noticed this one too, when I press the season selector, the number doesn't change; it just stays as season 1

**Comment by rhenwinch** [comment]
I see. Expect a fix on the next commits. Thanks for the heads up
